README.md

```diff
@@ -2,6 +2,8 @@ NUS ModBook
 ===========
 [![CI Status](https://github.com/AY2122S1-CS2103T-T13-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-T13-1/tp/actions)
 
-![Ui](docs/images/Ui.png)
+![Quick Start](docs/images/QuickStartView.png)
 
 NUS ModBook is a **desktop app for managing modules, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ModBook can help you organise modules faster than traditional GUI apps.
+
+Check out our [project website](https://ay2122s1-cs2103t-t13-1.github.io/tp/) to find out more!
```

Comment:
dont think this is needed? this is shown as the landing page of the website so may seem weird

Reply:
I think we should link to our docs somehow from this page?